setup.py

```diff
@@ -24,7 +24,7 @@
     long_description=long_description,
     long_description_content_type='text/markdown',
     install_requires=[
-        'dash>=0.40.0',
+        'dash==0.40.0',
```

Comment:
Why are we locking it to this version?

Reply:
For the same reason we've been holding off migrating to React 16 + Dash 0.42, i.e., tests don't pass, even if the actual apps work well:

* https://github.com/plotly/dash-bio/pull/344#issuecomment-490018688
* https://github.com/plotly/pytest-dash/pull/76

We locked this dependency a while ago:

* https://github.com/plotly/dash-bio/commit/7d4a1a238a2bc5fe604b5081d5846762e4bacf79
* https://github.com/plotly/dash-bio/commit/d093651162a1fb3a5103f9ea4866088c9dc55002

We'll be able to move forward as @byronz refactors `pytest-dash`.